ui/apps/ui/src/app/collections/data/all/adapter.data.ts

```diff
@@ -64,8 +64,18 @@ export const allCollectionsAdapter: IAdapter = {
       },
       {
         label: 'DOI',
-        value: data?.url || [],
-        filter: 'url',
+        value: data?.doi || [],
+        filter: 'doi',
+      },
+      {
+        label: 'Scientific domain',
```

Comment:
@agpul  `Scientific domain` is only for services! For OAG-type resources, there is `fos` (field of science) which looks similar at least to me. Please look at this filter [Explore](https://explore.openaire.eu/search/find?resultbestaccessright=%22Open%2520Access%22&active=result). Maybe we can present `Scientific domain` for services and `fos` for OAG? For trainings most likely there is nothing like this

Reply:
in the future, this field will be also valid for training, Scientific domain = FOS from my perspective, and in all catalogs it should be visible as one filter

Reply:
ok, so this should be done @danielkryska

Reply:
#279 - read and comment on what you think. I've created another ticket to resolve it as a new issue.

Reply:
ping @danielkryska